src/components/Pill/story.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import Pill from './';
+import Readme from './README.md';
+
+function handleClick(id) {
+  alert(`clicked ${id}`);
+}
+storiesOf('Uncategorized/Pill', module)
```

Comment:
This needs to be categorized. I'd put this under `Star System` since it incorporates a separate component and it's completely isolated

Reply:
I don't know the naming logic, sorry :)